.styleci.yml

```diff
@@ -0,0 +1,16 @@
+preset: psr2
+
+enabled:
+  - combine_consecutive_unsets
+  - newline_after_open_tag
+  - no_php4_constructor
+  - no_useless_else
+  - ordered_class_elements
+  - ordered_use
+  - strict
+  - strict_param
+  - php_unit_construct
+  - php_unit_strict
+finder:
+  name:
+    - "*.php"
```

Comment:
the finder is setup as so:

![image](https://cloud.githubusercontent.com/assets/2829600/21487801/418b54e0-cbcc-11e6-9ce3-6bb3a1e0fae1.png)